src/shared/components/query/QueryStore.ts

```diff
@@ -2275,6 +2294,13 @@ export class QueryStore {
     }
 }
 
+export enum Focus {
```

Comment:
this is too generic i think.  i would just keep safe and limit it to this feature